flowmachine/tests/server/test_action_handlers.py

```diff
@@ -9,10 +9,26 @@
     action_handler__get_geography,
     action_handler__get_query_params,
     action_handler__get_sql,
+    action_handler__run_query,
 )
 from flowmachine.core.server.zmq_helpers import ZMQReplyStatus
 
 
+def test_run_query_error_handled(monkeypatch, dummy_redis):
+    """
+    Run query handler should return an error status if query construction failed.
+    """
+    monkeypatch.setattr(Query, "redis", dummy_redis)
```

Comment:
It just occured to me: wouldn't it be simpler if we moved this monkeypatch into the fixture itself?